migration/Cargo.lock

```diff
@@ -3877,9 +4051,9 @@ checksum = "6e63cff320ae2c57904679ba7cb63280a3dc4613885beafb148ee7bf9aa9042d"
 
 [[package]]
 name = "spl-account-compression"
-version = "0.1.10"
+version = "0.1.5"
```

Comment:
I was actually having problems building locally and it was telling me `spl-account-compression` 0.1.5 was selected here but 0.1.8 was required elsewhere.

blockbuster 1.14 branch specifies:
spl-account-compression = { version = "0.1.5", features = ["no-entrypoint"] }

and bubblegum 1.14 branch specifies:
`spl-account-compression = { version="0.1.8", features = ["cpi"] }`

So theoretically it should be able to select something that works for both, but I was having issues until I deleted this Cargo.lock file and it chose 0.1.10 again.

Should we check in a new lock file here?  Or am I missing something, does `migration` build for you with this lock file?

Reply:
It did build but maybe something is wonky. It's tricky to line everything up when doing a large merge.

Reply:
It builds and runs for me after your update, thank you!